i18n/en.json

```diff
@@ -3237,7 +3237,7 @@
   },
   {
     "id": "api.templates.cloud_welcome_email.title",
-    "translation": "Your 30-day trial of the {{.WorkSpace}} workspace is ready to go!"
+    "translation": "Your workspace is ready to go!"
```

Comment:
nit: I assume there is no problem with keep passing the `Workspace` in https://github.com/mattermost/mattermost-server/blob/master/app/email/email.go#L318

Reply:
Nah but this is unnecessary, I'll drop it nice catch!